.github/workflows/release.yaml

```diff
@@ -33,45 +33,36 @@ jobs:
           go-version: 'stable'
 
       # This step is only used for testing in the official kangaroo repo
-      - name: Overwrite Names for release testing and fetch kando webhapp from github
-        if: ${{ github.repository }} == 'holochain-apps/holochain-kangaroo-electron'
+      - name: Overwrite Names for release testing
+        if: ${{ github.repository }} == 'holochain-apps/kangaroo-electron'
         run: |
           echo ${{ github.repository }}
           echo "overwriting names for release testing"
-          curl -f -L --output ./pouch/ziptest.webhapp https://github.com/holochain/ziptest/releases/download/ziptest-v0.1.0/ziptest.webhapp
```

Comment:
For kangaroo test releases on the main repo we're also going to use the `webhapp` field in `kangaroo.config.ts` going forward so we don't need to fetch it here anymore.